synapse/storage/databases/main/receipts.py

```diff
@@ -123,6 +123,15 @@ def f(txn):
             for row in rows
         }
 
+    async def get_linearized_receipts_for_all_rooms(
```

Comment:
It doesn't seem like this method provides and value and `_get_linearized_receipts_for_all_rooms` could just be renamed to this?

Reply:
Renamed.